lib/iex/lib/iex/helpers.ex

```diff
@@ -516,7 +516,7 @@ defmodule IEx.Helpers do
   [Recon](https://github.com/ferd/recon) project.
   """
   @doc since: "1.5.0"
-  def runtime_info(), do: runtime_info([:system, :memory, :limits])
+  def runtime_info(), do: runtime_info([:system, :memory, :limits, :allocators])
```

Comment:
```suggestion
  def runtime_info(), do: runtime_info([:system, :memory, :limits])
```

I would not include it by default, as it can be quite verbose. So let's remove it from here and let's add  a basic test that calling `runtime_info([:applications, :allocators])` do not crash. :)

Reply:
done